### ISSUE TITLE
How do we access the in preview feature : Install apps on behalf of users?

### ISSUE THREAD

**Issue by seanjb** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#4637]
Hi There, is there a way for us to access the "in preview" feature: Install apps on behalf of users?
It does not appear to be on our tennant?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6f91604a-35b0-dfbb-55b2-b601cb0841b0
* Version Independent ID: 84bcd0d7-b4ed-f13e-c84b-2ec6a7661ae9
* Content: [Manage app setup policies in Microsoft Teams - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies)
* Content Source: [Teams/teams-app-setup-policies.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/teams-app-setup-policies.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by scanum** [comment]
@seanjb thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by nikmanzotti** [comment]
I got here wondering the same thing. I have tried to search a reference to 'installed apps' in the Message Center, Microsoft 365 roadmap and Teams Tech Community, without any success.

I don't want to bother the support, which has been quite busy lately, with another request, so I've opened a post in the Tech Community. Let's see if some user knows something more about this!

https://techcommunity.microsoft.com/t5/microsoft-teams/install-apps-on-behalf-of-users-in-preview/m-p/1379608
